clang/test/CodeGenSYCL/wg_scope_var.cpp

```diff
@@ -0,0 +1,21 @@
+// RUN: %clang_cc1 -triple spir64-unknown-unknown-sycldevice -fsycl-is-device -disable-llvm-passes -I %S/Inputs -emit-llvm %s -o - | FileCheck %s
+
+// Checked that local variables declared by the user in PWFG scope are turned into globals in the local address space.
+// CHECK: @{{.*myLocal.*}} = addrspace(3) global i32 0
```

Comment:
FYI: I think the size of `int` depends on the host ABI, which is not set by the test, so on some platforms this check might fail due to `sizeof(int) != i32`. 
It's probably better to set aux-target-triple.